src/components/Modals/Send/views/Address.tsx

```diff
@@ -38,6 +44,9 @@ export const Address = () => {
   if (!(asset?.chain && asset?.name)) return null
 
   const adapter = chainAdapters.byChain(asset.chain)
+  const isEthereumChainAdapter = (
```

Comment:
This is needed because we instantiate adapter on the previous line, where tsc doesn't have any knowledge of the `ChainType` type of `asset.chain` yet, which means that `const validEnsAddress = await adapter.validateEnsAddress(value)` would fail at possibly undefined `validateEnsAddress` which is a property exclusive to the `ChainAdapter<ChainTypes.Ethereum>` type.

So in this case, it makes more sense to actually check for the adapter type than the chain type, both checks being effectively the same.